app/styles/layouts/editor.css

```diff
@@ -251,7 +251,19 @@
     line-height: 0;
 }
 
-
 .gh-editor-header-small .post-settings {
     padding: 13px 15px;
 }
+
+.gh-editor-wordcount {
+     position: fixed;
+     bottom: 0px;
+     padding:10px;
+ }
+
+ @media (max-width: 1200px) {
```

Comment:
The problem is that .gh-editor-header-small kicks in too late, the wordcount is already obscured by text at that point.

![resize over wordcount](https://cloud.githubusercontent.com/assets/73181/25361220/24ce28ea-29a1-11e7-9b19-687d11d714c1.gif)